src/EnergyPlus/ChillerGasAbsorption.cc

```diff
@@ -1722,9 +1722,17 @@ void GasAbsorberSpecs::calculateChiller(EnergyPlusData &state, Real64 &MyLoad)
             if (lCondWaterMassFlowRate > DataBranchAirLoopPlant::MassFlowTolerance) {
                 lCondSupplyTemp = lCondReturnTemp + lTowerLoad / (lCondWaterMassFlowRate * Cp_CD);
             } else {
-                ShowSevereError(state, "CalcGasAbsorberChillerModel: Condenser flow = 0, for Gas Absorber Chiller=" + this->Name);
-                ShowContinueErrorTimeStamp(state, "");
-                // ShowFatalError(state, "Program Terminates due to previous error condition.");
+                if (this->lCondWaterMassFlowRate_Index == 0) {
+                    ShowSevereError(state, "CalcGasAbsorberChillerModel: Condenser flow = 0, for Gas Absorber Chiller=" + this->Name);
+                    ShowContinueErrorTimeStamp(state, "");
+                    // ShowFatalError(state, "Program Terminates due to previous error condition.");
```

Comment:
I think this line can be deleted safely now, right?

Reply:
True. I think it was left as a comment from previous change somehow deliberately, considering that there are a few places in the code (also spotted in a couple places in the older code) that the developers massaged a little bit on whether to call the fatal error. So I would probably leave it as is, or a "special" note here.

Reply:
Got it. 👍🏽